src/styles/blocks/header.scss

```diff
@@ -0,0 +1,18 @@
+.header {
+  display: flex;
+  background: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__home-link {
+    width: 40px;
+    height: 40px;
```

Comment:
What about to create mixin for `square`? Can be used with `star` too.

Reply:
Not fixed

Reply:
What about to create mixin for square? Can be used with star too.

**It's the third time I'm asking you about this change.**